projects/js-packages/babel-plugin-replace-textdomain/.gitattributes

```diff
@@ -1,5 +1,4 @@
 # Files to exclude from the mirror repo
-.eslintignore     production-exclude
```

Comment:
In cases here and elsewhere where `.eslintignore` remains in the repo, wouldn't we want to keep them in the `.gitattributes`?

Reply:
I decided to add it to the monorepo-root `.gitattributes`, so we wouldn't have to remember to repeat it in all the projects.